boards/arm/nrf54l15pdk_nrf54l15/doc/index.rst

```diff
@@ -0,0 +1,136 @@
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
+
+.. figure:: img/nrf54l15pdk_nrf54l15.webp
+     :align: center
+     :alt: nRF54L15 PDK
+
+     nRF54L15 PDK (Credit: Nordic Semiconductor)
+
+Hardware
+********
+
+nRF54L15 PDK has two crystal oscillators:
+
+* High-frequency 32 MHz crystal oscillator (HFXO)
+* Low-frequency 32.768 kHz crystal oscillator (LFXO)
+
+The crystal oscillators can be configured to use either
+internal or external capacitors.
+
+Supported Features
+==================
+
+The nrf54l15pdk_nrf54l15 board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| SAADC     | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| RRAM      | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| TWIM      | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| GRTC      | on-chip    | counter              |
++-----------+------------+----------------------+
+| RTT       | Segger     | console              |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
++-----------+------------+----------------------+
+| UARTE     | on-chip    | serial               |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
+
+Other hardware features have not been enabled yet for this board.
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+To run a sample on the nRF54L15, you need to install all the
+required software.
+
+This example shows the :ref:`hello_world_user` sample,
+which uses the ``nrf54l15pdk_nrf54l15_cpuapp`` build target.
+
+To build and program the sample to the nRF54L15 PDK, complete the following steps:
+
+1. Connect the nRF54L15 PDK to you computer using the IMCU USB port on the PDK.
+#. Navigate to the :file:`zephyr/samples/hello_world` folder containing the sample.
+#. Build the sample by running the following command::
+
+      west build -b nrf54l15pdk_nrf54l15_cpuapp
```

Comment:
We typically tend to give out build instructions that are relative to the main zephyr/ folder. Also, please use `zephyr-app-commands` role instead of hard coding west build command. (you may take inspiration from other Nordic boards for consistency).